src/llm/apis/openai_completions.cpp

```diff
@@ -128,6 +131,57 @@ ov::Tensor load_image_stbi(const std::string& imageBytes) {
         SharedImageAllocator{image, desiredChannels, y, x});
 }
 
+static size_t writeMemoryCallback(void* contents, size_t size, size_t nmemb,
+    void* userp) {
+    size_t realsize = size * nmemb;
```

Comment:
There's a potential overflow here, but since it's handled by third party I'm not sure we do anything about it here (like return 0 if overflow?), maybe curl handles it somehow before passing the values to the callback.